android/fakedapp/src/main/java/com/solana/mobilewalletadapter/fakedapp/MainViewModel.kt

```diff
@@ -39,6 +37,13 @@ class MainViewModel(application: Application) : AndroidViewModel(application) {
     private val _uiState = MutableStateFlow(UiState())
     val uiState = _uiState.asStateFlow()
 
+    // should probably make an enum or similar abstraction but this is fine for now
```

Comment:
Since `fakedapp` is our most canonical example of how to use `clientlib`, I'd recommend we make an enum for this, and do the mapping in `MainActivity`. There's a decent chance that much of this sample gets copy-pasted elsewhere, so worth the extra verbosity to do it cleanly.

Reply:
makes sense, will do!